styleguide/derek/global-components/tracking/tracking.spec.js

```diff
@@ -228,6 +228,42 @@ describe('Zoolander Tracking Module', () => {
       expect(expected).to.eql(window.dataLayer.pop());
     });
 
+    it('should track internal nav clicks', () => {
+      const linkHtml = '<div id="main-navigation"><ul class="top-nav">' +
+          '<li class="item-i">Top level I</li> ' +
+          '<li class="item-ii"><a>Top level II</a>' +
+            '<ul class="mid-nav">' +
+              '<li class="item-a">Mid level I</li>' +
+              '<li class="item-b"><span>Mid level II</span>' +
+                '<ul class="third-nav">' +
+                  '<li class="item-1">Base I</li>' +
+                  '<li class="item-2"><a id="clickme">Base II</a></li>' +
+                  '<li class="item-3">Base III</li>' +
+                '</ul>' +
+              '</li>' +
+              '<li class="item-c">Mid level III</li>' +
+            '</ul>' +
+          '</li>' +
+          '<li class="item-iii">Top level III</li>' +
+        '</ul>' +
+      '</div>';
+
+      $('body').append(linkHtml);
+      Zoolander.Tracking.init();
+
+      $('#clickme').trigger('click');
```

Comment:
yes indeed. I still need to make sure this is bullet proof. Going to add a test for clicking on each "level" of the menu.